datafusion/core/src/physical_plan/mod.rs

```diff
@@ -135,13 +135,13 @@ pub trait ExecutionPlan: Debug + DisplayAs + Send + Sync {
     ///
     /// The default implementation returns `true` unless this operator
     /// has signalled it requires a single child input partition.
-    fn benefits_from_input_partitioning(&self) -> bool {
```

Comment:
Perhaps we can also update the documentation comment here to explain that the returned vector has an entry for each input

Reply:
Updated the documentation